lib/js-file.js

```diff
@@ -245,26 +184,55 @@ JsFile.prototype = {
     getLastNodeToken: function(node) {
         return this.getTokenByRangeEnd(node.range[1]);
     },
+
     /**
      * Returns the first token before the given.
      *
      * @param {Object} token
+     * @param {Boolean} [includeComments=false]
```

Comment:
I don't think we should have magic booleans. I think I'd much prefer this taking an options object.